improver_tests/acceptance/SHA256SUMS

```diff
@@ -317,6 +317,14 @@ fed1067d944506546b544aacf53fbc706bad700b8511b595d23a92b4803e0666  ./generate-top
 5df0093df31d08eb134797430672ac4815f339daab83e671582a6179d59f6413  ./generate-topography-bands-weights/multi_realization/input_land.nc
 8cd656273df9606cfbe0a25d20d0af6bbcd7731af456dffe1d7ba759fb2239a9  ./generate-topography-bands-weights/multi_realization/input_orog.nc
 328378a20cd3db75dc0e1286f8424a8fbecfaf14766106eb22e507fdaf4f8f17  ./generate-topography-bands-weights/multi_realization/kgo.nc
+59cceda783eab8dc5fc12a49a1351992b1cb0e02563af9aac11fed6474ea6eb0  ./hail-fraction/cloud_condensation_level.nc
+a7dd0bef17f317f7777f153fead177870e297b1c57f669be3beac09f81c02059  ./hail-fraction/convective_cloud_top_temperature.nc
+55d2f8487d92e8da2fbac095d5d6d8be4a0bfcaad5af74757450dec832163387  ./hail-fraction/hail_melting_level.nc
+2ee93c422480aabc40a78173016a56a177c5d581129ba30a79b45bba1f1a3ed1  ./hail-fraction/hail_size.nc
```

Comment:
No non-zero data in the hail size file.

Reply:
I've updated the acceptance test data to be for a case with a very small amount of non-zero hail size data.